fails:IO#tell raises IOError on closed stream
fail:IO#tell raises IOError on closed stream
